vellum/workflows/triage_message/nodes/triage_message_node.py

```diff
@@ -134,7 +157,11 @@ class TriageMessageNode(BaseInlinePromptNode):
 use text_reply to introduce yourself as Vargas JR: "Hi! My name is Vargas JR. I'm a fully automated \
 senior-level software developer, available for hire at a fraction of the cost of a full-time employee. \
 I can help with various software development tasks. How can I assist you today?" Make this response \
-engaging to get prospects more interested in learning more.""",
+engaging to get prospects more interested in learning more.
+
+When the message contains a scheduling link (cal.com, calendly.com, reclaim.ai, motion.com, etc.) or asks \
+you to schedule a meeting, use create_meeting with the scheduling link. The system will automatically book \
+the earliest available slot and then prompt you again to craft a response with the meeting details.""",
```

Comment:
We shouldn't need to update our system prompt bc we have an extensive tool description